docs/source/models.rst

```diff
@@ -130,11 +130,26 @@ hubert_ft_xlarge
 
 .. autofunction:: hubert_ft_xlarge
 
-.. currentmodule:: torchaudio.models.wav2vec2.utils
+Pre-trained Models
+------------------
+
+.. autoclass:: Wav2Vec2PretrainedModelBundle
+
+   .. automethod:: get_model
+
+   .. autoproperty:: labels
```

Comment:
can you check if this is the expected docs format you had in mind? https://357491-90321822-gh.circle-artifacts.com/0/docs/models.html#torchaudio.models.Wav2Vec2PretrainedModelBundle.labels

Reply:
This is what I had in mind, however, now I realize that it's missing type annotations. Let me work on it. Thanks for pointing.

Reply:
Updated https://357902-90321822-gh.circle-artifacts.com/0/docs/models.html#torchaudio.models.Wav2Vec2PretrainedModelBundle.labels